api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -493,15 +488,23 @@ private User initializeUserIfNeeded() {
   @Override
   public ResponseEntity<Profile> syncTrainingStatus() {
     User user = userProvider.get();
-    Profile profile = profileService.getProfile(user);
     try {
       userService.syncUserTraining(user);
     } catch(NotFoundException ex) {
       throw ex;
     } catch (ApiException e) {
       throw new ServerErrorException(e);
     }
-    return ResponseEntity.ok(profile);
+    return getProfileResponse(user);
+  }
+
+  @Override
+  public ResponseEntity<Profile> syncEraCommonsStatus() {
+    User user = userProvider.get();
+    NihStatus nihStatus = fireCloudService.getNihStatus();
+    userService.setEraCommonsStatus(nihStatus);
```

Comment:
nit: reassign the user w/ the return value from userService.setEraCommonsStatus?

user = userService.setEraCommonsStatus(nihStatus);